### PR TITLE
Adapt deliverable list to new Process requirements

### LINKED ISSUE #35: Adapt normative deliverable list to process 2017 requirements
See https://w3c.github.io/charter-drafts/charter-template.html#normative

### PR DESCRIPTION
**Pull request by xfq**
Fix #35.

PTAL @dontcallmedom

### CODE REVIEW

File: DeviceAPICharter.html [hunk 1]
```diff
@@ -131,29 +131,89 @@ <h3 id="rec-track">Normative Specifications</h3>
...
+    <p><b>Adopted Working Draft</b>: <a href="https://www.w3.org/TR/2017/WD-wake-lock-20170912/">Wake Lock API, W3C Working Draft 12 September 2017</a></p>
+    <p><b>Reference Draft</b>: <a href="https://www.w3.org/TR/2015/WD-wake-lock-20150212/">Wake Lock API, W3C First Public Working Draft 12 February 2015</a></p>
+    <p>Produced under <b>Working Group Charter:</b> <a href="https://www.w3.org/2011/07/DeviceAPICharter.html">2011-2016 Device APIs Working Group Charter</a></p>
+  </dd>
   <dt><a href="https://www.w3.org/TR/generic-sensor/">Generic Sensor API</a></dt>
-  <dd>An API that serves as basis for APIs that retrieve data from sensors</dd>
+  <dd>
+    <p>An API that serves as basis for APIs that retrieve data from sensors</p>
+    <p class="draft-status"><b>Draft state:</b> Working Draft</p>
+    <p><b>Adopted Working Draft</b>: <a href="https://www.w3.org/TR/2017/WD-generic-sensor-20171002/">Wake Lock API, W3C Working Draft 2 October 2017</a></p>
```
Comment (dontcallmedom) [inline DeviceAPICharter.html]: wrong spec title

File: DeviceAPICharter.html [hunk 2]
```diff
@@ -131,29 +131,89 @@ <h3 id="rec-track">Normative Specifications</h3>
...
+    <p><b>Reference Draft</b>: <a href="https://www.w3.org/TR/2016/WD-magnetometer-20160913/">Magnetometer Sensor, W3C First Public Working Draft 13 September 2016</a></p>
+    <p>Produced under <b>Working Group Charter:</b> <a href="https://www.w3.org/2016/03/device-sensors-wg-charter.html">2016-2017 Device and Sensors Working Group Charter</a></p>
+  </dd>
   <dt><a href="https://www.w3.org/TR/orientation-sensor/">Orientation Sensor</a></dt>
-  <dd>An API to monitor the device’s physical orientation in relation to a stationary three dimensional Cartesian coordinate system</dd>
+  <dd>
+    <p>An API to monitor the device’s physical orientation in relation to a stationary three dimensional Cartesian coordinate system</p>
+    <p class="draft-status"><b>Draft state:</b> Working Draft</p>
+    <p><b>Adopted Working Draft</b>: <a href="https://www.w3.org/TR/2017/WD-orientation-sensor-20170814/">Orientation Sensor, W3C Working Draft 14 August 2017</a></p>
+    <p><b>Reference Draft</b>: <a href="https://www.w3.org/TR/2017/WD-orientation-sensor-20170511/">Magnetometer Sensor, W3C First Public Working Draft 11 May 2017</a></p>
```
Comment (dontcallmedom) [inline DeviceAPICharter.html]: wrong spec title

**Review by dontcallmedom — COMMENTED** [review]
a couple of typos to fix, but otherwise lgtm

**Comment by xfq** [comment]
Fixed the typos. Thanks @dontcallmedom!

**Pull request merged**